### ISSUE TITLE
Confusion about step loss and value loss

### ISSUE THREAD

**Issue by tasmiaorni** [mfbalin/Concrete-Autoencoders#4]
Hello,
after each epoch the code prints:  **step - loss: 0.0043 - val_loss: 0.0043**. What does this step and value loss mean for each epoch??
Also at the starting of each epoch it shows : **mean max of probabilities: 6.54364e-05 - temperature 0.1.** what is mean max of probabilities  here??
Actually I am selecting 50 features using different  no of epochs and trying to find reconstruction error at each epoch. Which one is the reconstruction error ???
Thank you.

**Comment by mfbalin** [comment]
They are basically the training and validation loss values.

Mean max of probabilities is an indicator that shows whether the algorithm has converged. When it converges to 1, it means that it is safe to terminate.
